src/Symfony/Component/Validator/Resources/translations/validators.tr.xlf

```diff
@@ -402,6 +402,31 @@
                 <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
                 <target>Netmask'in değeri {{ min }} ve {{ max }} arasında olmaldır.</target>
             </trans-unit>
+            <trans-unit id="104">
+               <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+               <target>Dosya adı çok uzun. {{ filename_max_length }} karakter veya daha az olmalıdır.|Dosya adı çok uzun. {{ filename_max_length }} karakter veya daha az olmalıdır.</target>
+            </trans-unit>
+            <trans-unit id="105">
+              <source>The password strength is too low. Please use a stronger password.</source>
```

Comment:
The indentation is wrong, please use the same indentation from line no 406, thanks